File: tools/plotForce.m
clear; clf;
knp=dlmread('KnPdata.dat');
sim=dlmread('Results/r01forceHist.txt');
sim(1,:)=[];
plot(knp(:,1),knp(:,2),'ro');
hold on;
plot(sim(:,1)/16,sim(:,2),'b');
grid on;

File: mars_3000_0deg.case/plot_force.py
# Plot using gnuplot
set term dumb
plot 'Results/r01ForceNonDim.dat' u 1:2 w l
pause 1
reread

## Plot using matplotlib
# import matplotlib.pyplot as plt
# import numpy
# 
# # Get data points (timestep and Force)
# filename = 'Results/r01forceHist.txt'
# timestamp = []
# CT = []
# with open(filename) as forceFile:
#     # Skip first line
#     next(forceFile)
#     for line in forceFile:
#         # Handle empty lines and comments
#         if line and (line[0][0] != '#'):
#             data = line.split()
#             timestamp.append(float(data[0].strip()))
#             CT.append(float(data[1].strip()))
# 
# plt.plot(timestamp, CT, 'b')
# plt.xlabel('Iteration')
# plt.ylabel('CT')
# plt.grid()
# 
# plt.show(block=True)
# # input('Press Enter to exit')
